apps/emqx_bridge_s3/src/emqx_bridge_s3_upload.erl

```diff
@@ -73,49 +91,167 @@ fields(s3_upload_parameters) ->
                     }
                 )}
         ];
-fields(s3_action_resource_opts) ->
-    UnsupportedOpts = [batch_size, batch_time],
-    lists:filter(
-        fun({N, _}) -> not lists:member(N, UnsupportedOpts) end,
-        emqx_bridge_v2_schema:action_resource_opts_fields()
-    ).
+fields(s3_aggregated_upload_parameters) ->
+    lists:append([
+        [
+            {mode,
+                hoconsc:mk(
+                    aggregated,
+                    #{
+                        required => true,
+                        desc => ?DESC(s3_aggregated_upload_mode)
+                    }
+                )},
+            {container,
```

Comment:
A small suggestion, since this belongs to the new app (I noticed this after the last PR :sweat_smile: ): https://github.com/thalesmg/emqx/commit/a9244cbccf37bf191772299f1666d526cd72f26e

Reply:
Good point, but this commit is still part of unmerged PR, or am I missing something? What do you think if we merge this one as is to unblock the dashboard team sooner?